kong/cmd/utils/nginx_signals.lua

```diff
@@ -10,23 +11,23 @@ local nginx_search_paths = {
   "/usr/local/openresty/nginx/sbin",
   ""
 }
-local nginx_version_command = "-v"                            -- commandline param to get version
-local nginx_version_pattern = "^nginx.-openresty.-([%d%.]+)"  -- pattern to grab version from output
-local nginx_compatible = version.set("1.9.15.1")              -- compatible from-to versions
+local nginx_version_pattern = "^nginx.-openresty.-([%d%.]+)"
+local nginx_compatible = version.set(meta._DEPENDENCIES.nginx)
 
 local function is_openresty(bin_path)
-  local cmd = fmt("%s %s", bin_path, nginx_version_command)
+  local cmd = fmt("%s -v", bin_path)
   local ok, _, _, stderr = pl_utils.executeex(cmd)
+  log.debug("%s: '%s'", cmd, stderr:sub(1, -2))
   if ok and stderr then
-    log.debug("%s: '%s'", cmd, stderr:sub(1, -2))
     local version_match = stderr:match(nginx_version_pattern)
-    if (not version_match) or (not nginx_compatible:matches(version_match)) then
-      return nil, "incompatible nginx found. Kong requires OpenResty, version "..tostring(nginx_compatible) ..
-        (version_match and ", got "..version_match or "")
+    if not version_match or not nginx_compatible:matches(version_match) then
+      log.verbose("incompatible OpenResty found at %s. Kong requires version"..
+                  " %s, got %s", bin_path, version_match,
+                  tostring(nginx_compatible))
```

Comment:
missing a `return` here, it will now continue starting kong with an incompatible version